src/environment.d.ts

```diff
@@ -0,0 +1,9 @@
+namespace NodeJS {
+  interface ProcessEnv {
+    APPLICATION_ID: string;
+    DISCORD_TOKEN: string;
+    PUBLIC_KEY: string;
+    GITHUB_SECRET: string;
+    DISCORD_WEBHOOK: string;
+  }
+}
```

Comment:
this file is not needed as cloudflare is not supposed to access env vars from `process.env`, instead update the `Env` type at `src/index.ts`